build/scripts/restore.ps1

```diff
@@ -0,0 +1,13 @@
+param ([string]$nugetZipUrl = $(throw "Need an URL to the NuGet zip") )
+
+$destination = ${env:UserProfile}
+$outFilePath = [IO.Path]::ChangeExtension([IO.Path]::GetTempFileName(), "zip")
+write-host "Downloading $nugetZipUrl -> $outFilePath"
+$client = new-object System.Net.WebClient
+$client.DownloadFile($nugetZipUrl, $outFilePath)
+
+write-host "Extracting"
+Add-Type -assembly "System.IO.Compression.Filesystem"
+$archive = [IO.Compression.ZipFile]::OpenRead($outFilePath)
+rm -re (join-path $destination ".nuget\packages") -errorAction SilentlyContinue
```

Comment:
Wouldn't the former be better?  Or are you saying "it's ok and expected that we should not be able to remove this directory, and we want to keep on truckin gin that case?"